src/Geography.jl

```diff
@@ -1,3 +1,6 @@
+using DataDeps
+using Logging
+
 module Geography
 
 # Write your package code here.
```

Comment:
Here you need to add `include("download.jl")` to make the source code available in the package.